config/routes.rb

```diff
@@ -5,9 +5,10 @@
   resources :artworks, only: [:index, :show, :new, :create, :edit, :destroy] do
   resources :bookings, only: [:create]
   end
-  resource :dashboard, only: :show
+  resource :dashboard, only: :show # should it be resources with an s?
```

Comment:
Sy said when it is dashboard we are using resource and prefix becoming only dashboard
but they have another way to create dashboard that your making resources with name that you choice with 2 pages like:
# Option 1
# Non CRUD way, NOT RESTFUL
get '/requests', to: 'bookings#requests' # Offers made on my developer or artwork, on what other people have tried to book on your items, add functionality to either accept or reject


# get '/bookings', to: 'bookings#index'  bookings are on what you've tried to book,
resources :bookings, only: :index  # Bookings that I have made, maybe add a review functionality

# Option 2
# CRUD Way, RESTFUL
# DashboardsController would have 1 action, show
# dashboard, to: 'dashboards#show'
resource :dashboard, only: :show

# rails g controller dashboards